kvui.py

```diff
@@ -386,6 +386,60 @@ def insert_text(self, substring, from_undo=False):
         return super(ConnectBarTextInput, self).insert_text(s, from_undo=from_undo)
 
 
+def is_command_input(string: str) -> bool:
+    return string and string[0] in "/!"
```

Comment:
It would return empty string if the parameter is empty string - that could crash if someone is relying on `bool`
```suggestion
    return len(string) > 0 and string[0] in "/!"
```

Reply:
Didn't think it would be an issue as it's generally better to rely on truthiness than direct equality to `True`, but it also gets listed as a mypy issue so better to fix it to keep the typechecking error logs cleaner. Fixed.

Reply:
It's not just for comparing it for conditionals.
If a function returns `bool`, that tells me I can do this:
```python
count_command_inputs += is_command_input(c)
```
or this:
```python
bit_fields |= is_command_input(c) << 3
```